src/Control/Monad/Haskey.hs

```diff
@@ -42,6 +44,8 @@ import qualified Control.Monad.State.Lazy as StateL
 import qualified Control.Monad.State.Strict as StateS
 import qualified Control.Monad.Writer.Lazy as WriterL
 import qualified Control.Monad.Writer.Strict as WriterS
+import Control.Monad.Trans.Control
+import Control.Monad.Base
```

Comment:
Could you sort these imports and only import explicitly what you need, using the `import Module (...)` syntax